internal/command/group_grant.go

```diff
@@ -0,0 +1,449 @@
+	event, existingGroupGrant, err := c.removeGroupGrant(ctx, grantID, resourceOwner, false)
+	if err != nil {
+		return nil, err
+	}
+
+	pushedEvents, err := c.eventstore.Push(ctx, event)
+	if err != nil {
+		return nil, err
+	}
+	err = AppendAndReduce(existingGroupGrant, pushedEvents...)
+	if err != nil {
+		return nil, err
+	}
+	return writeModelToObjectDetails(&existingGroupGrant.WriteModel), nil
+}
+
+func (c *Commands) BulkRemoveGroupGrant(ctx context.Context, grantIDs []string, resourceOwner string) (err error) {
+	if len(grantIDs) == 0 {
+		return zerrors.ThrowInvalidArgument(nil, "COMMAND-8M0sd", "Errors.GroupGrant.IDMissing")
+	}
+	events := make([]eventstore.Command, len(grantIDs))
+	for i, grantID := range grantIDs {
+		event, _, err := c.removeGroupGrant(ctx, grantID, resourceOwner, false)
+		if err != nil {
+			return err
+		}
+		events[i] = event
+	}
+	_, err = c.eventstore.Push(ctx, events...)
+	return err
+}
+
+func (c *Commands) removeGroupGrant(ctx context.Context, grantID, resourceOwner string, cascade bool) (_ eventstore.Command, writeModel *GroupGrantWriteModel, err error) {
+	if grantID == "" {
+		return nil, nil, zerrors.ThrowInvalidArgument(nil, "COMMAND-I1qd5", "Errors.GroupGrant.IDMissing")
+	}
+
+	existingGroupGrant, err := c.groupGrantWriteModelByID(ctx, grantID, resourceOwner)
+	if err != nil {
+		return nil, nil, err
+	}
+	if existingGroupGrant.State == domain.GroupGrantStateUnspecified || existingGroupGrant.State == domain.GroupGrantStateRemoved {
+		return nil, nil, zerrors.ThrowNotFound(nil, "COMMAND-2Nz0t", "Errors.GroupGrant.NotFound")
+	}
+	if !cascade {
+		err = checkExplicitProjectPermission(ctx, existingGroupGrant.ProjectGrantID, existingGroupGrant.ProjectID)
+		if err != nil {
+			return nil, nil, err
+		}
+	}
+
+	removeGroupGrant := NewGroupGrantWriteModel(grantID, existingGroupGrant.ResourceOwner)
+	groupGrantAgg := GroupGrantAggregateFromWriteModel(&removeGroupGrant.WriteModel)
+	if cascade {
+		return groupgrant.NewGroupGrantCascadeRemovedEvent(
+			ctx,
+			groupGrantAgg,
+			existingGroupGrant.GroupID,
+			existingGroupGrant.ProjectID,
+			existingGroupGrant.ProjectGrantID), existingGroupGrant, nil
+	}
+	return groupgrant.NewGroupGrantRemovedEvent(
+		ctx,
+		groupGrantAgg,
+		existingGroupGrant.GroupID,
+		existingGroupGrant.ProjectID,
+		existingGroupGrant.ProjectGrantID), existingGroupGrant, nil
+}
+
+func (c *Commands) groupGrantWriteModelByID(ctx context.Context, groupGrantID, resourceOwner string) (writeModel *GroupGrantWriteModel, err error) {
+	ctx, span := tracing.NewSpan(ctx)
+	defer func() { span.EndWithError(err) }()
+
+	writeModel = NewGroupGrantWriteModel(groupGrantID, resourceOwner)
+	err = c.eventstore.FilterToQueryReducer(ctx, writeModel)
+	if err != nil {
+		return nil, err
+	}
+	return writeModel, nil
+}
+
+func (c *Commands) checkGroupGrantPreCondition(ctx context.Context, groupgrant *domain.GroupGrant, resourceOwner string) (err error) {
+	/* Why?
```

Comment:
This feature-flag provides you with the information if the fields can be used, should also be included in the group functions, not only for the group-grants.